verilog/analysis/checkers/token_stream_lint_rule_test.cc

```diff
@@ -55,6 +67,15 @@ TEST(StringLiteralConcatenationTest, FunctionFailures) {
         "fall.\""},
        ";",
        "\nendmodule"},
+      {"module m;\nstring x = 1 ?",
+       {kToken, "\"foo\\\nincorrect\""},
+       " : \"bar\";\nendmodule"},
```

Comment:
Another test where the second branch of the ?:  `"\"bar\\\nis incorrect\""` maybe ?

Reply:
Thanks for this test, I found one more bug when I applied the suggestion. The bug was related to incorrectly pointing token position.